app/views/bookings/show.html.erb

```diff
@@ -1,5 +1,20 @@
 <div class="container">
-  <h1>Booking for <%= link_to @booking.listing.title, listing_path(@booking.listing) %></span></h1>
-  <p>Date: <%= @booking.collection_date.strftime('%B %e, %Y') %></p>
-  <p>Customer: <%= @booking.user.first_name %> <%= @booking.user.last_name %></p>
```

Comment:
@sarhan-a I agree we need the business details on the confirmation page. Will make it more professional. I think It will be much easier if we have wireframe and follow a basic version of what we would like to achieve. Then we can work on the stylings. This is causing us to spend time on something that will be changed entirely later, which is a waste. Probably if you have something solid in your mind, you can assign this task to yourself to continue from here. Otherwise, we need a wireframe for sure. Let me know if you would like to create a wireframe so we can work on it together. Thanks

Reply:
I agree, but we can work with revisions, it doesn't have to be one version and final and we can still use your work going forward in updated versions.  Hindsight, maybe this is a page we don't see often as you only would see it if you bought something or booked something IRL, so if you want, let's first get that inspiration for a wireframe and checkin again later. Can you make the changes for the other aspects in the meantime and will merge and later versions with more data will ensue.